P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium.
+Dual-licensed under CC-by and BSD2 licences
+See the file COPYING.txt for details.
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<div xmlns="http://www.tei-c.org/ns/1.0" xmlns:xi="http://www.w3.org/2001/XInclude" type="div1" xml:id="CMC" xml:lang="en">
+  <head>Computer-mediated Communication</head>
+  <p>This chapter describes the TEI encoding mechanisms available for textual data which represents
+    discourse from genres of computer-mediated communication (CMC). It is intended to provide the
+    basic framework needed to encode CMC corpora.</p>
```

Comment:
Is it obvious what CMC contains and what not? Perhaps it might be helpful to the reader that e.g. E-Mails are not part of CMC and therefore not discussed here?

Reply:
@sabineseifert I think that the application of CMC to email is not certain, but possible--especially if we consider the archiving of en e-mail listserv. I am actually starting to propose something for the TEI conference to explore encoding between CMC and correspondence for email, connected to the TEI listserv archives. 

Yes, we should be clear about the scope of this chapter, but without absolutely cutting off some applications we are experimenting with. We should make this open for community experimentation with various forms of CMC.